src/VisualStudio/Core/Def/Implementation/Interop/Telemetry.cs

```diff
@@ -21,6 +21,8 @@ internal interface IVsTelemetrySession
         void PostDoublePiiProperty([ComAliasName("OLE.LPCOLESTR")]string szPropertyName, double varValue);
```

Comment:
what do you mean it doesn't match? if it didn't match, then nothing would have been working.... you found a serious bug!!! :) can you make it to exactly match the order of IDL?